src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/JudicialDecisionHandler.java

```diff
@@ -584,16 +611,43 @@ private String getJudgeHearingPrefType(CaseData caseData, YesOrNo isAppAndRespSa
                 .getDisplayedValue());
         }
 
+        if (caseData.getRespondentsResponses() != null && caseData.getRespondentsResponses().size() == 2) {
+            List<Element<GARespondentResponse>> respondentResponce = caseData.getRespondentsResponses();
+            String respondent1Id = caseData.getGeneralAppRespondentSolicitors().get(0).getValue().getId();
+            String respondent2Id = caseData.getGeneralAppRespondentSolicitors().get(1).getValue().getId();
+            if (respondentResponce != null) {
+                Optional<Element<GARespondentResponse>> respondenceElementOptional;
```

Comment:
can be respondentElementOptional1 and the other respondentElementOptional2. .
also 620 and 619 can be combined